config/default.yml

```
@@ -113,6 +113,13 @@ Performance/Detect:
   VersionAdded: '0.30'
   VersionChanged: '1.8'
 
+Performance/DeterministicRegexp:
+  Description: 'This cop identifies places where `split` argument can be replaced from a deterministic regexp to a string.'
+  SafeAutoCorrect: true
+  Enabled: true
+  VersionAdded: '1.8'
```

The next minor release is 1.9.

```suggestion
  VersionAdded: '1.9'
```